File: at-calc 1.0.0.cpp
#include <iostream>
#include <string>
#include <cmath>
using namespace std;

int main() {
	ant:
	long double x, y;
	string z;
	cout << "AT-Calc Version 1.0.0\nWrite first number.\n";
	cin >> x;
	cout << "Write second number.\n";
	cin >> y;
	cout << "Write which operation of those numbers you want to be performed\n(note that for mathematical operations which do not follow commutative law take it first to second number)\nor type IDK.\n";
	cin >> z;
	if (z == "IDK") {
		int r;
		cout << "Addition\nSubtraction\nMultiplication\nDivision\nModulus\nHypotenuse\nPower\nRoot\nLogarithm\nEnter one to reenter.\n";
		cin >> r;
		if (r == 1) {
			goto ant;
		}
		else {
			cat:
			int g;
			cout << "Wrong Input!!!\nEnter 1 to reenter.\n";
			cin >> g;
			if (g == 1) {
				goto ant;
			}
			else {
				cout << "Wrong input again!!!\n";
				goto cat;
			}
		}
	}
	else if (z == "Addition") {
		cout << x + y;
	}
	else if (z == "Subtraction") {
		cout << x - y;
	}
	else if (z == "Multiplication") {
		cout << x * y;
	}
	else if (z == "Division") {
		cout << x / y;
	}
	else if (z == "Hypotenuse") {
		cout << hypot(x, y);
	}
	else if (z == "Power") {
		cout << pow(x, y);
	}
	else if (z == "Root") {
		long double q;
		q = 1 / y;
		cout << pow(x, q);
	}
	else if (z == "Logarithm") {
		cout << log(y) / log(x);
	}
	else {
		dog:
		int s;
		cout << "Wrong Input!!!\nEnter 1 to reenter.\n";
		cin >> s;
		if (s == 1) {
			goto ant;
		}
		else {
			cout << "Wrong input again!!!\n";
			goto dog;
		}
	}
	return 0;
}

File: at-calc 1.0.0.py
#At-Calculator Version 1.0.0

import math

a = "fat"
while a == "fat":
    a == "on"
    print("AT-Calculator Version 1.0.0\nWrite first number.")
    x = float(input())
    print("Write second number.")
    y = float(input())
    print("Write which operation of those numbers you want to be performed\n(note that for mathematical operations which do not follow commutative law take it first to second number)\nor type IDK.")
    z = input()
    if z == "IDK":
        print("Addition\nSubtraction\nMultiplication\nDivision\nHypotenuse\nPower\nRoot\nLogarithm\nEnter one to reenter.")
        r = input()
        if r == "1":
            a = "fat"
        else:
            qan = "qut"
            while qan == "qut":
                print("Wrong Input!!!\nEnter 1 to reenter.")
                g = input()
                if g == 1:
                    qan = "and"
                else:
                    print("Wrong input again!!!\n")
                    qan = "qut"
    elif z == "Addition":
        print(x + y)
    elif z == "Subtraction":
        print(x - y)
    elif z == "Multiplication":
        print(x * y)
    elif z == "Division":
        print(x / y)
    elif z == "Power":
        print(x ** y)
    elif z == "Hypotenuse":
        print(math.hypot(x, y))
    elif z == "Root":
        q = 1 / y
        print(x ** q)
    elif z == "Logarithm":
        print(math.log(x, y))
    else:
        znat = "nat"
        while znat == "nat":
            print("Wrong Input\nEnter 1 to reenter.")
            sea = input()
            if sea == "1":
                znat = "q"
            else:
                print("Wrong input again!!!")
                znat == "nat"